components/fsxa/sections/Teaser.tsx

```diff
@@ -27,9 +27,27 @@ class TeaserSection extends FSXABaseSection<Payload> {
   render() {
     return (
       <Sections.TeaserSection
-        headline={(<FSXARichText content={this.payload.st_headline} />) as any}
-        kicker={this.payload.st_kicker}
-        text={(<FSXARichText content={this.payload.st_text} />) as any}
+        headline={
+          (
+            <FSXARichText
+              content={this.payload.st_headline}
+              editorName="st_headline"
```

Comment:
I assume this code will break unless the new version of the pattern library will be used as a dependency.